main.py

```diff
@@ -9,11 +10,13 @@
 
 import data_utils
 from model import Model
-from utils import log
-from utils.utils import ArgDict
+from utils import ArgDict
 from evaluate import evaluate, MultiLabelMetrics
 
 
+# logger = logging.getLogger(__name__)
```

Comment:
It would be useful to contain a timestamp in log messages.
Also, let's remove the logger name as we don't really change it.

```suggestion
# logger = logging.getLogger(__name__)
```
```suggestion
logging.basicConfig(level=logging.INFO, format='%(asctime)s %(levelname)s:%(message)s')
```